xml_converter/src/attribute/color.cpp

```diff
@@ -13,16 +14,67 @@
 
 using namespace std;
 
+////////////////////////////////////////////////////////////////////////////////
+// int_to_float // float_to_int
+//
+// Helper functions to convert the value of colors from one type to another.
+// Also serves to make sure the values stay within the bounds.
+////////////////////////////////////////////////////////////////////////////////
+float int_to_float(int input) {
```

Comment:
These function names should be more descriptive, they are not just converting int `1` into float `1.0` but instead an int in range 0-256, to an float in range 0.0-1.0